datasheet/Zubax_GNSS2_Datasheet.tex

```diff
@@ -0,0 +1,777 @@
+Default value: 100\\
+Units: $\text{Pascal}^2$
+
+\textbf{temp.variance}\\
+Temperature variance reported via \texttt{uavcan.equipment.air{\_}data.StaticAirData}.\\
+Default value: 4\\
+Units: $\text{Kelvin}^2$
+
+\textbf{mag.variance}\\
+Magnetic field variance reported via \texttt{uavcan.equipment.ahrs.Magnetometer}.\\
+Default value: 0.005\\
+Units: $\text{Gauss}^2$
+
+\textbf{mag.scaling{\_}coef}\\
+New in firmware v3.1.\\
+Magnetic field vector scale. Some UAV autopilot systems (PX4, APM) do not correclty handle magnetic field measurements made in highly magnetically distorted environments. It is possible to work around these issues by reducing the magnitude of the magnetic field vector.\\
+For PX4 and APM users: if your autopilot reports that magnetometer offsets are too high, reduce this parameter to approximately 0.6 and recalibrate the magnetometer.\\
+Default value: 1.0, i.e. no rescaling.
+
+\textbf{gnss.warn{\_}dimens}\\
+Set the node status to WARNING if the number of dimensions in the GNSS solution is below this threshold. Values: 1 - time-only fix, 2 - 2D fix, 3 - 3D fix.\\
+Default value: 0
+
+\textbf{gnss.warn{\_}sats}\\
+Set the node status to WARNING if the number of satellites used in the GNSS solution is below this threshold.\\
+Default value: 0
+
+\textbf{nmea.uart{\_}on}\\
+Enable NMEA output via DroneCode port (UART). This configuration parameter does not affect NMEA output over USB.\\
+Default value: 0 (disabled)
+
+\textbf{mag.pwron{\_}slftst}\\
+New in firmware v4.0.\\
+Perform power on self test of the magnetometer when booting. Note that the manufacturer of the magnetometer recommends to hold the sensor stationary during the self test procedure. Therefore it is recommended to disable this feature unless it is expected that the board will be always held stationary during powering on. Otherwise the self test may be occasionally failing, delaying the proper start up of the device.\\
+Default value: 1 (enabled)
+
+\textbf{gnss.old{\_}fix{\_}msg}\\
+New in firmware v4.0.\\
+Broadcast the deprecated message \texttt{uavcan.equipment.gnss.Fix} alongside the new message\\ \texttt{uavcan.equipment.gnss.Fix2}.\\This feature is enabled by default for compatibility purposes; however, it is recommended to disable it in order to reduce the CAN bus traffic if it is not needed.\\
+Default value: 1 (compatibility mode)
+
+\chapter{Identification information}
+
+This section documents the device properties that are reported in response to identification requests: the CLI command \textbf{zubax{\_}id} and the UAVCAN service\texttt{uavcan.protocol.GetNodeInfo}.
+\begin{itemize}
+\item Product ID and UAVCAN node name: \textbf{com.zubax.gnss}.
+\item Hardware version: 2.x.
+\end{itemize}
+
+\chapter{Firmware update}
+Instructions below are only applicable to Zubax GNSS v2.2 and newer.\\
+It is recommended to use Zubax Toolbox or the UAVCAN GUI Tool for updating the firmware.\\
```

Comment:
You're abusing forced newlines a lot, please don't do that.

In typography, a paragraph conveys one simple finished idea. If your idea is finished here, start a new paragraph. If not, keep writing in the same one.

A forced newline is a dirty hack used to fight LaTeX when it won't behave, and it should never be used unless you have a very, very solid reason for it. I strongly suspect that this whole document can be written without forced newlines at all. For example, the Sapog reference manual is written with zero forced newlines, and it's a large document.